docs/advanced/general-privacy.md

```diff
@@ -0,0 +1,63 @@
+It's also important to note, that different wallets might use different [address types](address-details). This can also give a hint about your identity, which combined with other signals can ultimately reveal your identity.
+
+## Trade-offs of reusing vs not reusing addresses
+
+For privacy, the best practice is to not reuse an address ever. A cryptocurrency address can be thought of as a cryptocurrency invoice instead. The term "address" does not mean, that it was designed to reuse it, like one would do with a bank account number for example. If someone would really care about privacy he or she would use a new receiving address even on a website, generated by a script. 
+On the other hand, address reuse is preferred in situations when privacy is not important and we don't want to make the mistake of sending over large amount of funds to a wrong address. Reusing addresses is also preferred when transparency is important.
+For POS voting, not using the same address is not possible yet. 
+
+## Change address
+
+If we only spend fraction of an [UTXO](https://coincentral.com/utxo-beginners-explainer/), a change output will be generated. This is not good for privacy. For example, the change UTXO will be merged with another unspent transaction output next time when we are spending money, thus creating a signal that both of the transactions belong to us. An average wallet software won’t give us information about this, we won’t know, for example, that 2 UTXOs were merged to create the transaction. If you care about privacy, you should exactly know which UTXOs are being spent. [This](https://medium.com/@nopara73/coin-control-is-must-learn-if-you-care-about-your-privacy-in-bitcoin-33b9a5f224a2) is a good article on coin control.
+
+## Avoid identification when acquiring cryptocurrencies 
+
+Try to avoid getting cryptocurrencies from websites that require KYC or AML, or any other means of identification. Be aware that centralized exchanges can be hacked, thus providing the collected information to the hacker, who can belong to any kind of organization. It’s also possible that at some point an oppressive government will request the collected information. ATMs can be equipped with cameras, or there can be cameras nearby the ATM. This combined with the blockchain can reveal our identity. 
+
+## Avoid leaving identifiers in public places
+
+Many times this is something physical, for example leaving behind a printed QR code somewhere, writing our nickname or real name on a piece of paper. Your name can be inprinted on the next page of the sticky notes stack, for example. Or that could also happen, that you leave behind your phone or computer unlocked for some minutes, with important information displayed on screen.
+
+## Centralized mixers
```

Comment:
This section seems to be completely focused on bitcoin. Can we please either make this Decred-centric, or remove it.

Reply:
I included these paragraph to make the reader think about privacy tech. I was not trying to direct the reader to start using these tools, I included them for historic reasons mainly. I know that it is a bit focused on bitcoin, but I don't know how otherwise to make the reader think about the basic concept.

I can try to re-frame this paragraph, but the basic intention is not to give present-day information in this section. CoinShuffle++ was meant to be present day information.

Reply:
I shortened the Centralized Mixers section